platform/mellanox/mlnx-onie-fw-update.sh

```diff
@@ -2,6 +2,7 @@
 
 # Copyright (C) 2019 Mellanox Technologies Ltd.
 # Copyright (C) 2019 Michael Shych <michaelsh@mellanox.com>
+# Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
i think it should be 2019-2021 instead of 3 lines

Reply:
Fixed